### PR TITLE
Translate docs/tasks/traffic-management/app-health-check/index.md into chinese

### PR DESCRIPTION
**Pull request by xiaosuiba**
Translate docs/tasks/traffic-management/app-health-check/index.md into chinese.

### CODE REVIEW

File: content_zh/docs/tasks/traffic-management/app-health-check/index.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+---
+title: Istio Service 健康检查
+description: 展示如何对 Istio service 进行健康检查.
```
Comment (rootsongjc) [inline content_zh/docs/tasks/traffic-management/app-health-check/index.md]: 最后的"."要改成句号

File: content_zh/docs/tasks/traffic-management/app-health-check/index.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+在 Kubernetes 中存在三种 liveness 和 readiness 探针：1）命令; 2）http 请求; 3）tcp 请求。在这项任务中，我们在启用和禁用 Istio mutual TLS 时，分别提供了使用前两种选项的示例。
+
+## 开始之前
+
+* 了解 [Kubernetes liveness 和 readiness 探针](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/)，Istio [认证策略](/docs/concepts/security/authn-policy/) 和 [双向 TLS 认证](/docs/concepts/security/mutual-tls/) 的概念。
+
+* 有一个安装了 Istio 的 Kubernetes 集群，但没有启用全局双向 TLS（例如按照 [安装步骤](/docs/setup/kubernetes/quick-start/#installation-steps) 中的描述使用 istio-demo.yaml，或者在使用 [Helm](/docs/setup/kubernetes/helm-install/) 时将 `global.mtls.enabled` 设置为 false）。
+
+## 使用命令选项的 Liveness 和 readiness 探针
```
Comment (rootsongjc) [inline content_zh/docs/tasks/traffic-management/app-health-check/index.md]: 保持标题的大小写统一

**Comment by geeknoid** [comment]
Please update this PR to use the new annotations for code blocks. Instead of backticks, you need to use

{{< text bash >}}

Please see [here](https://preliminary.istio.io/about/contribute/writing-a-new-topic/#embedding-preformatted-blocks) for info

Thanks.

**Comment by rootsongjc** [comment]
@xiaosuiba CI failed, please fix it.
```
- ./public/blog/2018/egress-tcp/index.html
  *  External link https://dev.mysql.com/doc/refman/5.7/en/mysqlsh.html failed: 404 No error
- ./public/zh/docs/tasks/traffic-management/app-health-check/index.html
  *  3:85: ERROR: Unexpected end tag : p (line 3)
htmlproofer 3.9.1 | Error:  HTML-Proofer found 2 failures!
```

**Pull request merged**